pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -294,7 +294,9 @@ func (m *manager) reconcileState() (success []reconciledContainer, failure []rec
 }
 
 func findContainerIDByName(status *v1.PodStatus, name string) (string, error) {
-	for _, container := range status.ContainerStatuses {
+	allStatuses := status.InitContainerStatuses
```

Comment:
Nit: `allStatuses` feels like a slightly confusing variable name when we explicitly assign it to `status.InitContainerStatuses`. I know its only for one line, so definitely not a big deal... but I wonder if we could find a better name.

Reply:
I was mimicking the naming from above:
https://github.com/kubernetes/kubernetes/pull/77162/files#diff-67c46e7cbb41ea5e675000a3298e6b32L226

I'd be happy to change it though if you like. Maybe just `statuses`?